static/code/svg.js

```diff
@@ -80,6 +87,7 @@ svgEditor.resize = function() {
  */
 svgEditor.setString = function(xmlString) {
   svgEditor.canvas.setSvgString(xmlString);
```

Comment:
Presumably `xmlString` is expected to include the `<svg>` wrapper?  Might be good in general to document which XML source strings are and are not expected to include this.